lua/astrocommunity/completion/blink-cmp-tmux/init.lua

```diff
@@ -0,0 +1,16 @@
+return {
+  "saghen/blink.cmp",
+  dependencies = { "mgalliou/blink-cmp-tmux" },
+  opts = function(_, opts)
+    opts.sources = vim.tbl_deep_extend("keep", opts.sources or {}, {
+      default = {},
+      providers = {
+        tmux = {
+          module = "blink-cmp-tmux",
+          name = "tmux",
+        },
+      },
+    })
+    table.insert(opts.sources.default, "tmux")
+  end,
+}
```

Comment:
```suggestion
return {
  "mgalliou/blink-cmp-tmux",
  lazy = true,
  specs = {
    {
      "Saghen/blink.cmp",
      optional = true,
      opts = {
        sources = {
          default = { "tmux" },
          providers = {
            tmux = { name = "tmux", module = "blink-cmp-tmux" },
          },
        },
      },
    },
  },
}
```

This can be simplified to this which (1) is clearer to read and (2) makes it easier for the user to disable conditionally in their user configuration from the `plugins/` side.